crowdfunding_compassion/templates/participant_page.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Couldn't you reuse the project template and inject different values in variables ? This way we would avoid too much code duplication and the need to change both templates the day we want to make some design changes.

Reply:
yes